src/status_im/component_test_helpers.clj

```diff
@@ -0,0 +1,17 @@
+(ns status-im.component-test-helpers (:refer-clojure :exclude [test]))
```

Comment:
can you please move these out of src.status_im?
imo we should never be importing/requiring from status_im (or status_im2 into quo2.

Can we just put this in src/something. Preferably lets keep test utilities separate from app utils.
e. src/test-utils 🤔

Reply:
Also I have a pr in the works where I have added some component-tests for pieces in status_im2 so it would be good to have this at the root level of the codebase 👍

Reply:
For sure, I just opted to put it in the same directory where the other `test_helpers.cljs` already lives.

The question is where do we put it? Any recommendation that you think other people would generally like?

Reply:
src/test-utils or src/test-helpers imo

Reply:
Good suggestion. I'm going for `src/test-helpers/component.cljs` (and the `clj` counterpart). In a follow-up PR I could move the existing `status-im.test-helpers` to `src/test-helpers/unit.cljs`.

Reply:
FYI: I've moved the file in this commit https://github.com/status-im/status-mobile/pull/14650/commits/7069a7c4ab0792d52e6097fd11f7d699ad68eb37. As I mentioned, I'll move the other test helper in a separate PR. Thanks for the feedback.